src/modules/TalentProfileModule/Section/talentExperience.tsx

```
@@ -0,0 +1,78 @@
+import { useEffect, useState } from 'react';
+import { Button } from '@/components';
+import Experience, { ExperienceDetail } from '@/components/ui/experience';
+import UserProfileCard, { UserProfile } from '@/components/ui/profile';
```

ESLint

[error] 'UserProfileCard' is defined but never used.

(@typescript-eslint/no-unused-vars)